extensions/Worker.Extensions.SendGrid/SendGridAttribute.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Attribute used to bind a parameter to a SendGridMessage that will automatically be

```

Comment:
This description doesn't apply as we won't support the binding to `SendGridMessage`